cirq/protocols/circuit_diagram_info_protocol.py

```diff
@@ -153,19 +153,6 @@ def format_real(self, val: Union[sympy.Basic, int, float]) -> str:
             return str(val)
         return f'{float(val):.{self.precision}}'
 
-    def format_complex(self,
```

Comment:
We shouldn't remove this formatter from the diagrams. Why do you think it needs to go?